tools/topology/sof-bdw-codec.m4

```diff
@@ -73,14 +73,14 @@ SectionGraph."PIPE_NAME" {
 DAI_ADD(sof/pipe-dai-playback.m4,
 	1, SSP, 0, Codec,
 	PIPELINE_SOURCE_1, 2, s24le,
-	1000, 0, 0)
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
 
 # capture DAI is SSP0 using 2 periods
 # Buffers use s24le format, 1000us deadline on core 0 with priority 0
 DAI_ADD(sof/pipe-dai-capture.m4,
 	2, SSP, 0, Codec,
 	PIPELINE_SINK_2, 2, s24le,
-	1000, 0, 0)
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
```

Comment:
probably better to keep DMAs on broadwell, this is difficult enough with DMAs alone and IIRC there are no external timers?